web/styles/right_sidebar.css

```diff
@@ -13,20 +13,39 @@ $user_status_emoji_width: 24px;
     overflow: auto;
 }
 
-#user_presences {
-    max-width: 95%;
+.toggle-narrow-users,
+.toggle-other-users {
+    width: 7px;
+}
+
+#narrow-users-presence-container,
+#other-users-presence-container {
+    &.no-display {
+        display: none;
+    }
+}
+
+#narrow-user-presences,
+#other-user-presences {
+    max-width: 90%;
+    margin-left: 10px;
     overflow-x: hidden;
     list-style-position: inside; /* Draw the bullets inside our box */
 
+    &.collapsed {
+        display: none;
+    }
+
     & li {
         overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
         list-style-type: none;
         border-radius: 4px;
-        padding-right: 15px;
-        padding-top: 1px;
-        padding-bottom: 2px;
+        /* Override .empty-list-message !important styling */
```

Comment:
I don't like this, but I also don't have context on why we decided to use `!important` for the main empty list message.

I'm also assuming we don't want to use that styling here, but maybe we do?

Reply:
with the override:

<img width="292" alt="image" src="https://github.com/zulip/zulip/assets/5634097/a72cfe49-3129-4e66-8609-548b5d2ceecc">

without:

<img width="321" alt="image" src="https://github.com/zulip/zulip/assets/5634097/47a5df54-10e7-4032-9d99-82e5da92f4a1">